hw7/7.1/tree.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+
+typedef struct Node Node;
+
+// ������ ���������� NULL
+Node* createTree();
+// ������� ������ ���������� ��� ������
+void freeTree(Node* root);
```

Comment:
```suggestion
// просто возвращает NULL
Node* createTree();

// очищает память выделенную под дерево
void freeTree(Node* root);
```
Вообще, функции с комментариями отделяются пустой строкой друг от друга